PPF-300 - Add setup playwright

simon-debruijn opened the pull request:
### Added
- Add playwright setup with basic auth

---
Ticket: https://jira.uitdatabank.be/browse/PPF-300

```
Running 1 test using 1 worker
  1 passed (9.9s)
```

Related appconfig PR:
https://github.com/cultuurnet/appconfig/pull/577

simon-debruijn commented on playwright.config.ts:
```diff
@@ -0,0 +1,33 @@
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
```
Kristof mentioned junit output can be formatted nicely. SO we could change the output if needed later

simon-debruijn commented:
You can run the tests by doing `npx playwright test` in the root of the project. I'll add scripts for it in package.json and the makefile

simon-debruijn commented:
I'm using the credentials from `Dev End2End Test ACC` that are stored in bitwarden

LucWollants reviewed (commented):
A few questions/remarks:
- Can it also be added as a `make` command to the `Makefile`?
- Not sure if it needs to be added to the docs?
- It also requires a change to the `.ENV` file?
- Should the `baseURL` be configurable?

simon-debruijn commented:
> A few questions/remarks:
> 
> * Can it also be added as a `make` command to the `Makefile`?
> * Not sure if it needs to be added to the docs?
> * It also requires a change to the `.ENV` file?
> * Should the `baseURL` be configurable?

I've addressed all your remarks

simon-debruijn commented on README.md:
````diff
@@ -221,12 +221,32 @@ $ make stan
+$ make test-e2e-filter filter=view-integrations-in-overview.test.ts
+```
+
+- Run an e2e filtered on the test title
````
```suggestion
- Run an e2e test filtered on the test title
```
